tests/test_hexary_trie.py

```diff
@@ -316,3 +338,12 @@ def test_hexary_trie_missing_node():
 
     # Other keys are still accessible
     assert trie.get(key2) == b'val2'
+
+
+def test_hexary_trie_raises_on_pruning_snapshot():
+    db = {}
+    trie = HexaryTrie(db, prune=True)
```

Comment:
Pretty minor stray copy-paste. Could combine previous lines since we don't need later access to `db`. I'll save that fix until after review.